backend-shared/config/db/util/utils.go

```diff
@@ -34,7 +34,7 @@ func GetGitOpsEngineSingleInstanceNamespace() string {
 }
 
 func GetOrCreateManagedEnvironmentByNamespaceUID(ctx context.Context, workspaceNamespace v1.Namespace,
-	dbq db.DatabaseQueries, log logr.Logger) (*db.ManagedEnvironment, error) {
+	dbq db.DatabaseQueries, log logr.Logger) (*db.ManagedEnvironment, error, bool) {
```

Comment:
Can you add a quick comment like this: "the bool return value is true if the managedenvironment was created; false otherwise."
(that way it's obvious what the bool return value is for, for the next person to look at this function)